lib/dmv.rb

```diff
@@ -1,16 +1,26 @@
 class Dmv
-
+attr_reader :facilities, :services
   def initialize
     @facilities = []
+    @services = []
   end
 
   def add_facility(facility)
     @facilities << facility
   end
 
+  def import_facility(api_data)
+    api_data.each do |data|
+      facility = Facility.new(data)
+      @facilities << facility
+    end
+    @facilities
+  end
+
   def facilities_offering_service(service)
-    @facilities.find do |facility|
+    @facilities.select do |facility|
       facility.services.include?(service)
     end
   end
 end
+
```

Comment:
Great job with this class. I think having this `import_facility` method makes a lot of sense here. I think I still would have preferred to see this behavior in a separate class as we did with the `VehicleFactory`.

The only other thing would be to make sure to require the `Facility` class at the top of this file since you are instantiating the facility objects from within the file inside the class.